### PR TITLE
refactor(console): extract common i18n values

### PR DESCRIPTION
**Pull request by xiaoyijun**
## Summary

* Extract commonly used i18n value into `general`.
* Move `general` into `admin_console` in locale file.

### Todo
- Align I18n Key Scope in Admin Console (LOG-3104)

## Testing

Tested in the admin console.

### CODE REVIEW

File: packages/console/src/components/Table/TableError.tsx [hunk 1]
```diff
@@ -31,7 +31,7 @@ const TableError = ({ title, content, onRetry, columns }: Props) => {
           <div className={styles.content}>
             {content ?? t('admin_console.errors.unknown_server_error')}
           </div>
-          {onRetry && <Button title="general.retry" onClick={onRetry} />}
+          {onRetry && <Button title="admin_console.general.retry" onClick={onRetry} />}
```
Comment (demonzoo) [inline packages/console/src/components/Table/TableError.tsx:34]: Hmmm, a common issue in this PR. I think now we all use `admin_console` prefix, am I right? Then we can eliminate the usage of `I18nKey` and use `AdminConsoleKey` instead. Also use `admin_console` as the keyPrefix when calling `useTranslation`.
Then we don't have to add `admin_console` to each phrase any more.

Comment (gao-sun) [reply packages/console/src/components/Table/TableError.tsx:34]: agreed

Comment (xiaoyijun) [reply packages/console/src/components/Table/TableError.tsx:34]: This is in another issue: [LOG-3104](https://linear.app/silverhand/issue/LOG-3104)

Comment (demonzoo) [reply packages/console/src/components/Table/TableError.tsx:34]: ok

File: packages/phrases/src/locales/en.ts [hunk 2]
```diff
@@ -1,31 +1,35 @@
...
-    set_up: 'Set up',
-    customize: 'Customize',
-  },
   admin_console: {
     title: 'Admin Console',
     sign_out: 'Sign Out',
     profile: 'Profile',
     admin_user: 'Admin',
     system_app: 'System',
+    general: {
```
Comment (demonzoo) [inline packages/phrases/src/locales/en.ts:9]: I think some of the phrases above can also be put into `general`, at least `sign_out`, `profile`, `admin`

File: packages/phrases/src/locales/zh-cn.ts [hunk 3]
```diff
@@ -205,22 +189,17 @@ const translation = {
       back_to_connectors: '返回连接器',
       check_readme: '查看 README',
       tab_settings: '设置',
```
Comment (demonzoo) [inline packages/phrases/src/locales/zh-cn.ts:191]: I saw numerous `设置`s in this file. Can we go through the document again and also extract these common words into the `general` category?

File: packages/phrases/src/locales/zh-cn.ts [hunk 4]
```diff
@@ -269,12 +247,8 @@ const translation = {
       created_password: '初始密码：',
       created_button_close: '关闭',
       created_button_copy: '复制',
```
Comment (demonzoo) [inline packages/phrases/src/locales/zh-cn.ts:249]: These two can be put in `general` as well

File: packages/phrases/src/locales/en.ts (hunk 2 above)
Comment (xiaoyijun) [reply packages/phrases/src/locales/en.ts:9]: They are not shared between pages. I prefer to keep them out of `general`.

File: packages/phrases/src/locales/zh-cn.ts (hunk 3 above)
Comment (xiaoyijun) [reply packages/phrases/src/locales/zh-cn.ts:191]: I found it is not suitable for being extracted.
The `Settings` tab name on the menu bar is not the same concept as the `Settings` sub-page under the application details page.

File: packages/phrases/src/locales/zh-cn.ts (hunk 4 above)
Comment (xiaoyijun) [reply packages/phrases/src/locales/zh-cn.ts]: updated

File: packages/phrases/src/locales/zh-cn.ts (hunk 3 above)
Comment (demonzoo) [reply packages/phrases/src/locales/zh-cn.ts:188]: But if you search the entire document, you'll find 8 results of '设置', at least some of them CAN be extracted into general, right?

**Review by demonzoo — APPROVED** [review]

**Comment by wangsijie** [comment]
refactor(console)?

File: packages/phrases/src/locales/zh-cn.ts (hunk 3 above)
Comment (xiaoyijun) [reply packages/phrases/src/locales/zh-cn.ts]: Yeah, updated~

**Pull request merged**